components/Buttons/src/MDCFloatingButton.h

```diff
@@ -37,7 +37,15 @@ typedef NS_ENUM(NSInteger, MDCFloatingButtonShape) {
   /**
    A 56-point circular button surrounding a 36-point square icon or short text.
    */
-  MDCFloatingButtonShapeLargeIcon = 2
+  MDCFloatingButtonShapeLargeIcon = 2,
+  /**
+   An Extended FAB with a 24-point square icon on the leading side of the text.
```

Comment:
If this supports dynamic type, note how scaling works.  Perhaps something along the lines of:
An extended FAB with a square icon (default 24 x 24) on the leading edge of the text.